compiler/cpp/src/thrift/generate/t_swift_generator.cc

```diff
@@ -95,19 +110,20 @@ class t_swift_generator : public t_oop_generator {
   void generate_xception(t_struct* txception);
   void generate_service(t_service* tservice);
 
-  void print_const_value(ostream& out,
-                         string name,
-                         t_type* type,
-                         t_const_value* value,
-                         bool defval = false,
-                         bool is_property = false);
+  // void print_const_value(ostream& out,
+  //                        string name,
+  //                        t_type* type,
+  //                        t_const_value* value,
+  //                        bool defval = false,
+  //                        bool is_property = false);
```

Comment:
Can you remove commented out code, if useless ? Otherwise please put a comment as to why we need to keep it as a comment.